src/XMTP.types.ts

```diff
@@ -1,7 +1,28 @@
+import { Compression } from "./XMTP.enums";
+import { ContentTypeID } from "./lib/ContentTypeID";
+
 export type ChangeEventPayload = {
   value: string;
 };
 
 export type XMTPViewProps = {
   name: string;
 };
+
+/*
+ * EncodedContent
+ *
+ * Represents encoded content and its metadata.
+ *
+ * @param {ContentTypeID | {}} type - The content type ID or object for this content.
+ * @param {string} content - The encoded content data.
+ * @param {Compression} compression - The compression algorithm used, if any.
+ * @param {string} fallback - A fallback representation of the content, if any.
+ */
+
+export type EncodedContent = {
+  type: ContentTypeID | {};
```

Comment:
When would this just be an empty object?

Reply:
Good catch, it would have been a possible state before I removed the `EncodedContent` class but I missed removing it here. Just updated that. Thanks!